src/v2/Apps/Order/Components/__tests__/SavedAddresses.jest.tsx

```diff
@@ -4,12 +4,51 @@ import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
 import { Button } from "@artsy/palette"
 import { AddressModal } from "v2/Apps/Order/Components/AddressModal"
 import React from "react"
+import { createTestEnv } from "v2/DevTools/createTestEnv"
+import { RootTestPage } from "v2/DevTools/RootTestPage"
+import { userAddressMutation } from "v2/Apps/__tests__/Fixtures/Order/MutationResults"
+import { SavedAddressItem } from "v2/Apps/Order/Components/SavedAddressItem"
 
 jest.unmock("react-relay")
 
-beforeEach(() => {
-  afterEach(() => {
-    jest.clearAllMocks()
+class SavedAddressesTestPage extends RootTestPage {
+  async selectEdit() {
+    this.find(`[data-test="addressModal"]`).simulate("click")
+    await this.update()
+  }
+}
+
+describe("Saved Addresses mutations", () => {
+  const { mutations, buildPage } = createTestEnv({
+    Component: (props: any) => (
+      <SavedAddressesFragmentContainer inCollectorProfile {...props} />
+    ),
+    defaultData: userAddressMutation,
+    TestPage: SavedAddressesTestPage,
+    query: graphql`
+      query SavedAddresses_Test_Query {
```

Comment:
Just a reminder that these query names have to be globally unique:

![](https://static.damonzucconi.com/_capture/ksz5HguA.png)

Reply:
@dzucconi Thanks for raising! It should be [fixed now](https://github.com/artsy/force/pull/7133) in master (I was also investigating a MP failure that led to this).